libcaf_core/caf/actor_cast.hpp

```diff
@@ -150,7 +150,7 @@ class actor_cast_access<To, From, neutral_cast> {
 template <class T, class U>
 T actor_cast(U&& what) {
   using from_type =
-    typename std::remove_const<typename std::remove_reference<U>::type>::type;
+    typename std::remove_const<typename std::remove_reference_t<U>>::type;
```

Comment:
No longer having to write `typename` in most cases is actually the best thing about the `_t` aliases.

```suggestion
    std::remove_const_t<std::remove_reference_t<U>>;
```

Reply:
Also, please add a comment here that this code should use `remove_cvref` when switching to C++20.